addons/event/views/event_views.xml

```diff
@@ -142,21 +150,24 @@
             <field name="name">event.type.search</field>
             <field name="model">event.type</field>
             <field name="arch" type="xml">
-                <search string="Event Categories">
+                <search string="Event Templates">
                     <field name="name"/>
                     <filter string="Online Event" name="online_event" help="Online Event" domain="[('is_online', '=', True)]"/>
                 </search>
             </field>
         </record>
 
         <record model="ir.actions.act_window" id="action_event_type">
-            <field name="name">Event Categories</field>
+            <field name="name">Event Templates</field>
             <field name="res_model">event.type</field>
         </record>
 
         <record id="menu_event_type" model="ir.ui.menu">
             <field name="action" ref="event.action_event_type"/>
         </record>
+        <menuitem id="menu_event_configuration" name="Configuration" parent="event_main_menu"
+            sequence="100"/>
+        <menuitem name="Event Templates" id="menu_event_type" action="action_event_type" parent="menu_event_configuration"/>
```

Comment:
Ensure all menus are defined in menu_views, and updated with the correct action when the action is defined. That way we have a unique file for menu structure.